docker/shadow-hand/kinetic/base/Dockerfile

```diff
@@ -5,12 +5,15 @@ LABEL Description="This ROS Kinetic image contains Shadow's dexterous hand softw
 ENV remote_shell_script="https://raw.githubusercontent.com/shadow-robot/sr-build-tools/$toolset_branch/ansible/deploy.sh"
 
 ENV PROJECTS_WS=/home/user/projects/shadow_robot
-RUN echo "Running one-liner" && \
-    apt-get update && \
+
+RUN echo "Running one-liner" && apt-get update && \
     wget -O /tmp/oneliner "$( echo "$remote_shell_script" | sed 's/#/%23/g' )" && \
     chmod 755 /tmp/oneliner && \
     gosu $MY_USERNAME /tmp/oneliner -w $PROJECTS_WS/base -r sr-build-tools -b $toolset_branch -i data/shadow_robot-kinetic.rosinstall -v "kinetic" -t mongodb,pyassimp && \
 
+    echo "Installing production tools" && \
+    curl https://raw.githubusercontent.com/shadow-robot/sr-build-tools/$(echo $toolset_branch | sed 's/#/%23/g')/bin/install-production-tools.sh | bash && \
```

Comment:
please user this approach for one liner call http://shadow-robot.readthedocs.io/en/latest/generated/shadow_robot/INSTALL.html#ros-kinetic